QA.md

```diff
@@ -18,7 +18,7 @@ python gradio_chat.py --baseonly
 
 No, you can just run with the default settings and our current dataset in the project to test. If you want you can also pick your own dataset but you will need to tweak some setting see [this](walkthrough-hf-dataset.md) tutorial for more info.
 
-### Windows AI Studio Preview would not start
+### AI Toolkit Preview would not start
```

Comment:
We move the prerequisite check from beginning to model fine tunning, so here the question can be changed to "AI Toolkit Preview would not scaffold the fine tunning project".

Reply:
updated title